src/show.cpp

```diff
@@ -35,6 +35,21 @@ void Show::print()
     }
 }
 
+void Show::getVLANIfaces(std::set<std::string>& vlanIfaces)
+{
+    for (const auto& it : netObjects)
+    {
+        if (it.second.find(Dbus::ethInterface) != it.second.end())
+        {
+            const auto cfgVlan = getProperties(static_cast<std::string>(it.first).c_str(), Dbus::vlanInterface);
```

Comment:
I little bit confused by `static_cast<std::string>(it.first)`.
I won't claim that it's wrong, but it seems to me `it.first.str` significantly shorter and more human readable.

Reply:
> I little bit confused by `static_cast<std::string>(it.first)`. I won't claim that it's wrong, but it seems to me `it.first.str` significantly shorter and more human readable.

Yes, this one is better.